apps/hpc-ftsadmin/src/app/utils/table-headers.ts

```diff
@@ -0,0 +1,501 @@
+import { LanguageKey, t } from '../../i18n';
+import { Strings } from '../../i18n/iface';
+import { Query } from '../components/tables/table-utils';
+import { FilterKeys } from './parse-filters';
+
+/** Declare which tables there can be */
+export type TableType = 'flows' | 'organizations' | 'keywords';
+
+/** The nomenclature to define these IDs is to write it like: name of the DB table, and after the dot, the propperty.
+ *  if it's not a DB field, just write down the name
+ */
```

Comment:
Not sure why Prettier doesn't enforce length of lines in docstrings and proper spacing (to avoid double spaces), but this looks so messy. Also, you've got the typo in there.
```suggestion
/**
 * The nomenclature to define these IDs is to write it like:
 * name of the DB table, and after the dot, the property.
 * If it's not a DB field, just write down the name.
 */
```